EncoreTickets.SDK.Tests/UnitTests/Basket/Extensions/ReservationExtensionTests.cs

```diff
@@ -97,4 +107,45 @@ public void TotalFaceInShopperCurrency_Correct()
             AssertExtension.AreObjectsValuesEqual(DefaultPrice.MultiplyByNumber(DefaultQuantity), result);
         }
     }
+
+    public static class ReservationExtensionTestsSource
```

Comment:
I do not know why TestCaseSource is difficult to maintain, for me this is much simpler when a test depends on the source object. The current test depends on the original reservation object, and we can pass the object with different properties or, for example, it can be null. Such different objects do not require special test logic. And if something changes in the implementation of the method, we just need to update the source reservations for the test and not change the test logic

Reply:
If you think that it's best to use TestCaseSource here that's fine. But I still believe that they are difficult to maintain since they tend to be longer than actual tests (and in this case would be longer than the same test with a bunch of TestCase attributes and creation of the reservation object inside the test itself) and they are disconnected from the rest of the test logic

Reply:
This is not logic, TestCaseSource is needed for easy initialization of test data. And I think you can easily jump to it from the test using IDE shortcuts without much effort. Let's leave it as is here :)